ddsc/cmdparser.py

```diff
@@ -8,6 +8,7 @@
 
 DESCRIPTION_STR = "DukeDSClient ({}) Manage projects/folders/files in the duke-data-service"
 INVALID_PATH_CHARS = (':', '/', '\\')
+EXISTING_FILES_FULL_COMPARE_HELP = "Perform full comparision when determining if a file needs to be uploaded again."
```

Comment:
The text here should help the user make a decision about using the feature - including what the default behavior is, when it's good enough, and in what cases (likely rare) they might want to enable it. It's a feature we think most users should not use, so we want to communicate that too - and reassure that we're still comparing files on name/size and computing checksums prior to transmission.

After seeing the logic implemented , another idea would be to avoid the true/false option here (and a hidden default), and instead make the max size be a CLI option, e.g.

```
--compare-checksum-size-limit: Default 10MB. Prior to upload, changes to local files are 
detected by comparing file name and size. If both name and size match, and the file is below
 this limit, checksums are computed as a last resort. If file is above this limit, modifications 
that do not change the size of the file will not be detected for uploads.

Checksums take more time to compute on larger files, but the size may be more likely to 
change. If file modifications not detected for upload, consider raising this limit.
```

That way we can highlight why we chose what we did and give a clear path to change it if necessary.
We'd still have to have a way to specify __no limit__, but I'll put this out for debate first 😄

Reply:
This `--compare-checksum-size-limit` flag sounds good to me. I'll tweak the code to use this flag.